### ISSUE TITLE
Global usemin disable target

### ISSUE THREAD

**Issue by dlmiles** [yeoman/grunt-usemin#475]
It would be very useful to disable usemin completely with a setting like:

```javascript
  options: { disable:true }
```

Without needed to completely reconfigure Gruntfile.js to come up with a new rule scheme to perform the following tasks that are still useful.

Enabling the disable:true option would still configure concat/uglify/cssmin but would setup a pass-through mode.  usemin will still however resolve and fix as necessary relative paths like:

```html
  <!-- build:js ../scripts/vendor/modernizr.js -->
  <script src="../bower_components/modernizr/modernizr.js"></script>
  <!-- endbuild -->
```

In the above example usemin would:

* Ignore the target path "../scripts/vendor/modernizr.js" set in the HTML comment
* Transform the src="" attribute by removing the relative component outside the tree of the target directory the HTML file is in.  So this means removal of "../bower_components/" part.  It should check based on the build:js file type (js in this case), make a new output path: "js/modernizr/modernizr.js" using the remaining.  If really necessary to ensure path uniqueness "js/usemin-disabled/modernizr/modernizr.js" and error on any duplicate paths created in processing.
* Configure "concat" to basically file copy, from the source location to the new location of a single file at a time, nothing is concatentated.
* Configure "uglify" to do nothing.
* Configure "cssmin" to do nothing.
* The final pass of "usemin" will only edit the src="" locations, and remove the 2 HTML comment markers.  It does not replace the multiple HTML elements with a single (concatentated element), it only correct the src="" attribute to the new path now inside the project of the copied file.

This allows an easy way at the flick of a switch to not transform anything, but still fixup and populate all JS and CSS assets from outside the tree.

This facilitated development/debugging where the goal is easy to read source code in the original distribution layouts and filenames, using verbatim copies.  Where we do not care about download overhead or hits to server performance concerns.  Since we are developing.

Then to go back to production testing / releasing we flick the switch and retest using minimized copies.

**Comment by stephanebachelier** [comment]
@dlmiles it should be covered by #487. Marked as a duplicate.

This should be opened until we merged #487 and document your use case.

**Comment by stephanebachelier** [comment]
@dlmiles I close it in favor of #487 and #456.
